libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F412Cx.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/env python
+'''
+these tables are generated from the STM32 datasheet RM0402 in en.DM00180369.pdf for the
+STM32F412 and STM32CubeMX configuration tool
+'''
+
+# additional build information for ChibiOS
+build = {
+    "CHIBIOS_STARTUP_MK"  : "os/common/startup/ARMCMx/compilers/GCC/mk/startup_stm32f4xx.mk",
+    "CHIBIOS_PLATFORM_MK" : "os/hal/ports/STM32/STM32F4xx/platform.mk"
+}
+
+# MCU parameters
+mcu = {
+    # location of MCU serial number
+    'UDID_START' : 0x1FFF7A10,
```

Comment:
this should come from the chibios headers now

Reply:
Fixed by removing the file